misk/web/tabs/config/src/sagas/configSaga.ts

```diff
@@ -14,6 +14,8 @@ import {
   CONFIG, dispatchConfig, IAction, IActionType
 } from "../actions"
 
+const dateFormat = "YYYY-MM-DD HH:mm:ss"
```

Comment:
YASSS my fave

Reply:
Presumably this is local time? Doesn’t matter for this use case, but sometimes we’re mixing UTC and local and it’s clumsy. I have no solutions for that

Reply:
I put this in the demo just to visually show that the Config shown on screen was updated as of a certain point. We should definitely put this in UTC to match existing paradigm. I think DayJS can do this. #427